README.md

```diff
@@ -101,6 +101,13 @@ The chip-HAL crates currently support the following peripherals:
   - [x] I2C using `TWI`
   - [x] SPI
   - [x] ADC
+* [`atmega168-hal`](./chips/atmega168-hal)
+  - [x] Spinning Delay
+  - [x] `PORTB`, `PORTC`, `PORTD` as digital IO (v2)
+  - [x] `USART0` for serial communication
+  - [x] I2C using `TWI`
+  - [x] SPI
+  - [x] ADC
```

Comment:
I've done some changes to the README layout in commit 1e52121ff490 ("README: Cleanup document structure").  Please adjust this accordingly.  Also, please add a bullet-point (further down) for the board you're adding.

Reply:
I will fix this.